index.html

```diff
@@ -79,118 +80,8 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
 </div>
       </section>
       <section class="project" id="portfolio">
-        <div class="project-content">
-          <h3 class="underlined">My Recent Works <span class="underlined-hr"></span> </h3>
-        <div class="project-list">
-          <div class="project-item">
-            <div class="project-item-img">
-          <img src="/img/tonic2.png" alt="" class="img">
-        </div>
-      </div>
-      <div class="project-disc">
-      <h2 class="project-heading">Multi-Post Stories</h2>
-      <p class="project-disc-main">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standar dummy text.</p>
-
-      <ul class="project-btns">
-        <li><button type="button" class="project-btns-list">css</button></li>
-        <li><button type="button" class="project-btns-list">html</button></li>
-        <li><button type="button" class="project-btns-list" id="bts">bootstrap</button></li>
-        <li><button type="button" class="project-btns-list">Ruby</button></li>
-        
-      </ul>
-      <div class="project-bttn">
-        <button type="button" class="project-btn">See Project</button>
-      </div>
-      </div>
-          </div>
-        </div>
-    <div class="all-project">
-      <div class="projects-primary">
-        <div class="projects-heading1">
-          <h3>Professional art <span>printing Data</span></p>
-        </div>
-        <div class="projects-disc1">
-          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
-        </div>
-        <ul class="project1-bttns">
-          <li><button type="button" class="project1-btns">html</button></li>
```

Comment:
As the previous reviewer requested please stick to the design provided and make sure all popups are working properly.

![Screen Recording 2023-02-11 at 11 03 58 PM](https://user-images.githubusercontent.com/25887253/218279318-fc476e92-16d7-4518-9e4a-e0b7d402eae2.gif)